content/documentation/4-batch-developer-guides/2-batch/6-standalone-simple-task-kubernetes.md

```diff
@@ -0,0 +1,134 @@
+---
+path: 'batch-developer-guides/batch/standalone-simple-task-kubernetes/'
+title: 'Deploying a task application on Kubernetes'
+description: 'Guide to deploying spring-cloud-stream-task applications on Kubernetes'
+---
+
+# Deploying a task application in Kubernetes
+
+This guide will walk you through how to deploy and run a simple [spring-cloud-task](https://spring.io/projects/spring-cloud-task) application on Kubernetes.
+
+We will deploy the sample [billsetuptask]() application to Kubernetes.
+
+## Setting up the Kubernetes cluster
+
+For this we need a running [Kubernetes cluster](/documentation/installation/kubernetes/). Here we will deploy to `minikube`.
+
+### Verify minikube is up and running:
+
+```bash
+$ minikube status
+
+host: Running
+kubelet: Running
+apiserver: Running
+kubectl: Correctly Configured: pointing to minikube-vm at 192.168.99.100
+```
+
+### Install the database
+
+We will install a MySQL server, using the default configuration from Spring Cloud Dataflow. Execute the following commands:
+
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/src/kubernetes/mysql/mysql-deployment.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/src/kubernetes/mysql/mysql-pvc.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/src/kubernetes/mysql/mysql-secrets.yaml
+
+$ kubectl apply -f https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/src/kubernetes/mysql/mysql-svc.yaml
+```
+
+### Build a Docker image for the sample task application
+
+We will build the `billsetuptask` app, which is configured with the [jib maven plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#build-your-image):
+
+Clone the task samples git repo, and cd to the `billsetuptask` directory.
+
+```bash
+$ eval $(minikube docker-env)
+$ mvn clean package jib:dockerBuild
+```
+
+This will add the image to the `minikube` Docker registry.
+Verify its presence by finding `springcloudtask/billsetuptask` in the list of images:
+
+```bash
+$ docker images
+```
+
+### Deploy the app
+
+The simplest way to deploy a task application is as a standalone [Pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/). Deploying tasks as a [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) or [CronJob](https://kubernetes.io/docs/tasks/job/) is considered best practice for production environments, but is beyond the scope of this guide.
+
+Save the following to `task-app.yaml`
```

Comment:
I'm thinking we will want to add this to our billsetuptask project.   And add a link to it from this document (while still keeping the content here in the document).